docs/docs/references/configuration/config-file.md

```diff
@@ -271,6 +382,10 @@ misconfiguration:
   # Same as '--include-non-failures'
   # Default is false
   include-non-failures: false
+
+  # Same as '--policy-bundle-repository'
+  # Default is 'ghcr.io/aquasecurity/trivy-policies:0'
+  policy-bundle-repository: ghcr.io/aquasecurity/trivy-policies:0  
```

Comment:
The bundle is already live at the new URL so we can use it. Once [this PR](https://github.com/aquasecurity/trivy/pull/6476) is merged we can also change the flag to the new one. 

```suggestion

  # Same as '--policy-bundle-repository'
  # Default is 'ghcr.io/aquasecurity/trivy-checks:0'
  policy-bundle-repository: ghcr.io/aquasecurity/trivy-checks:0  
```